nextflow.config

```diff
@@ -1,56 +1,60 @@
 params {
 
-    outdir = "/hps/nobackup/rdf/metagenomics/service-team/users/chrisata/taxdb_generation_v6"
-    dummy_fasta = "/hps/software/users/rdf/metagenomics/service-team/users/chrisata/taxdb_generation_nf/assets/dummy.fasta"
-    empty_file = "/hps/software/users/rdf/metagenomics/service-team/users/chrisata/taxdb_generation_nf/assets/EMPTY.txt"
+    outdir           = "/hps/nobackup/rdf/metagenomics/service-team/users/chrisata/taxdb_generation_v6"
```

Comment:
we should probably just make this null and have the user specify it as an input parameter